code/modules/mob/living/basic/space_fauna/giant_spider/spider_variants.dm

```diff
@@ -0,0 +1,329 @@
+ */
+/mob/living/basic/giant_spider/nurse/scrawny
+	name = "scrawny nurse spider"
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	health = 30
+	maxHealth = 30
+	desc = "Furry and black, it makes you shudder to look at it. This one has brilliant green eyes, and looks abnormally thin and frail."
+	menu_description = "Weaker version of the nurse spider, specializing in healing their brethren and placing webbings very swiftly, but has very low amount of health and deals low damage."
+
+/**
+ * # Flesh Spider
+ *
+ * A subtype of giant spider which only occurs from changelings.
+ * Has the base stats of a hunter, but they can heal themselves and spin webs faster.
+ * They also occasionally leave puddles of blood when they walk around. Flavorful!
+ */
+/mob/living/basic/giant_spider/hunter/flesh
+	desc = "A odd fleshy creature in the shape of a spider.  Its eyes are pitch black and soulless."
+	icon_state = "flesh_spider"
+	icon_living = "flesh_spider"
+	icon_dead = "flesh_spider_dead"
+	web_speed = 0.7
+	menu_description = "Self-sufficient spider variant capable of healing themselves and producing webbbing fast, but has less health and damage."
+
+/mob/living/basic/giant_spider/hunter/flesh/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/blood_walk, \
+		blood_type = /obj/effect/decal/cleanable/blood/bubblegum, \
+		blood_spawn_chance = 5)
+	// It might be easier and more fitting to just replace this with Regenerator
+	AddElement(/datum/component/healing_touch,\
+		heal_brute = maxHealth * 0.5,\
+		heal_burn = maxHealth * 0.5,\
+		self_targetting = HEALING_TOUCH_SELF_ONLY,\
+		interaction_key = DOAFTER_SOURCE_SPIDER,\
+		valid_targets_typecache = typecacheof(list(/mob/living/basic/giant_spider/hunter/flesh)),\
+		extra_checks = CALLBACK(src, PROC_REF(can_mend)),\
+		action_text = "%SOURCE% begins mending themselves...",\
+		complete_text = "%SOURCE%'s wounds mend together.",\
+	)
+
+/// Prevent you from healing other flesh spiders, or healing when on fire
+/mob/living/basic/giant_spider/hunter/flesh/proc/can_mend(mob/living/source, mob/living/target)
+	if (on_fire)
+		balloon_alert(src, "on fire!")
+		return FALSE
+	return TRUE
+
+/**
+ * # Viper Spider (Wizard)
+ *
+ * A spider form for wizards. Has the viper spider's extreme speed and strong venom, with additional health and vent crawling abilities.
+ */
+/mob/living/basic/giant_spider/viper/wizard
+	maxHealth = 80
+	health = 80
+	menu_description = "Stronger assassin spider variant with an unmatched speed, high amount of health and very deadly poison, but deals very low amount of damage. It also has ability to ventcrawl."
+	apply_spider_antag = FALSE
+
+/mob/living/basic/giant_spider/viper/wizard/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+
+/**
+ * # Sergeant Araneus
+ *
+ * Once a bat, now finally a spider. This friendly arachnid hangs out in the HoS office on some space stations.
```

Comment:
```suggestion
 * This friendly arachnid hangs out in the HoS office on some space stations.
```

I know it's funny to reference old-code slipups like this but comments should be pertinent to current code.